railties/test/application/configuration_test.rb

```diff
@@ -94,14 +94,18 @@ def suppress_default_config
     def restore_default_config
       FileUtils.rm_rf("#{app_path}/config/environments")
       FileUtils.mv("#{app_path}/config/__environments__", "#{app_path}/config/environments")
+      remove_from_env_config "production", "config.log_level = :error"
     end
 
     def suppress_sqlite3_warning
-      add_to_config "config.active_record.sqlite3_production_warning = false"
+      add_to_config <<-RUBY
+        config.active_record.sqlite3_production_warning = false
+      RUBY
```

Comment:
What is the a reason to change this?

Reply:
This was rebased into a single commit in 3e2fc8e2fb7e6c18b75d2a64162273b2a2f3e250, but the goal was to maintain consistent indentation like other examples of `add_to_config`.

For example, when using `add_to_config` without the `<<-`:

```
puts File.read("#{app_path}/config/application.rb")
require_relative "boot"

require "rails/all"

# Require the gems listed in Gemfile, including any gems
# you've limited to :test, :development, or :production.
%w(sprockets/railtie importmap-rails).each { |r| require r }

module AppTemplate
  class Application < Rails::Application
    # Initialize configuration defaults for originally generated Rails version.
    config.load_defaults 7.2

    # Please, add to the `ignore` list any other `lib` subdirectories that do
    # not contain `.rb` files, or that should not be reloaded or eager loaded.
    # Common ones are `templates`, `generators`, or `middleware`, for example.
    config.autoload_lib(ignore: %w(assets tasks))

    # Configuration for the application, engines, and railties goes here.
    #
    # These settings can be overridden in specific environments using the files
    # in config/environments, which are processed later.
    #
    # config.time_zone = "Central Time (US & Canada)"
    # config.eager_load_paths << Rails.root.join("extras")
        config.hosts << proc { true }
        config.eager_load = false
        config.session_store :cookie_store, key: "_myapp_session"
        config.cache_store = :mem_cache_store
        config.active_support.deprecation = :log
        config.action_controller.allow_forgery_protection = false
        config.active_record.sqlite3_production_warning = false
        config.active_support.cache_format_version = 7.1
config.active_record.sqlite3_production_warning = false
config.logger = MyLogRecorder.new




  end
end
```

Reply:
This commit can also be dropped if #49715 is merged

Reply:
Droppedl